src/fsharp/service/FSharpCheckerResults.fs

```diff
@@ -2234,6 +2236,139 @@ type FSharpCheckProjectResults
 
         FSharpAssemblyContents(tcGlobals, thisCcu, Some ccuSig, tcImports, mimpls)
 
+    member _.TryEmitMetadataOnlyAssembly(stream: Stream) =
```

Comment:
This is almost the same logic as what is in `fsc`. We should find a way to share instead of duplicating.